mods/ra/rules/vehicles.yaml

```diff
@@ -714,6 +714,12 @@ QTNK:
 	SelectionDecorations:
 		VisualBounds: 44,38,0,-4
 	MadTank:
+	Detonates:
+		ChargeDelay: 96
+		DetonationDelay: 42
+		ChargeSound: MADCHRG2.AUD
+		DetonationSound: MADEXPLO.AUD
```

Comment:
It's not. It was lowercase in the MadTank trait you're removing. That's your change.